internal/type_conversions.go

```diff
@@ -53,6 +53,12 @@ func ToFloat64(value interface{}) (float64, error) {
 		return strconv.ParseFloat(string(v), 64)
 	case fmt.Stringer:
 		return strconv.ParseFloat(v.String(), 64)
+	case bool:
+		bv := float64(0)
+		if v {
+			bv = float64(1)
+		}
+		return bv, nil
```

Comment:
This is simpler I think
```suggestion
		if v {
			return float64(1), nil
		}
		return float64(0), nil
```